spec/models/member_spec.rb

```diff
@@ -160,4 +163,165 @@
       it { is_expected.not_to be_thredded_admin }
     end
   end
+
+	describe "renewed_subscription" do
+
+			before(:each) do
+				@member = FactoryBot.create(:member)
+			end
+
+		context "when is the first subscription" do
```

Comment:
Do you mean "when a member subscribes for the first time" ?
In that case, it could be useful to show this in the test setup, in that example group